pkg/spiffecert/spiffecertfakes/fake_reloader.go

```diff
@@ -0,0 +1,102 @@
+// Code generated by counterfeiter. DO NOT EDIT.
+package spiffecertfakes
```

Comment:
Could we store all these fakes in their own `fake` directory?

Reply:
I could look into the configuring the counterfeiter package, but what good would that be? its all autogenerated code in this package wouldnt you prefer it to be held in the same place?

Reply:
Oh, I didn't even realize these were already in their own directory. Nevermind!